### PR TITLE
Repo and package renaming

### PR DESCRIPTION
**Pull request by okwolf**
Also bumping version to be more in line with Hyperapp.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
-# @hyperapp/logger
+# `hyperapp-logger`
```
Comment (jorgebucaran) [inline README.md]: #  Hyperapp Logger

Comment (jorgebucaran) [reply README.md]: What do you think?

Comment (jorgebucaran) [reply README.md]: https://gist.github.com/JorgeBucaran/533f75c2330cf79afc2e622ad7a07e84

Comment (okwolf) [reply README.md]: #lgtm

File: package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+  "name": "hyperapp-logger",
+  "version": "1.0.1",
+  "description": "Log Hyperapp state updates and action information to the console",
   "main": "dist/logger.js",
   "jsnext:main": "src/index.js",
   "module": "src/index.js",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/hyperapp/logger.git"
+    "url": "git+https://github.com/hyperappJS/hyperapp-logger.git"
```
Comment (frenzzy) [inline package.json]: here and below `hyperappJS` => `hyperapp`?

Comment (jorgebucaran) [reply package.json]: 👍

**Comment by jorgebucaran** [comment]
@okwolf You could also remove this line:

```
"jsnext:main": "src/index.js",
```

as it's no longer relevant. You can look it up or I will later  if I remember.

File: package.json (hunk 2 above)
Comment (okwolf) [reply package.json]: That org is so 8 hours ago...

**Comment by okwolf** [comment]
I think this looks good to merge and republish the package now. Unless you're planning on changing the org again in the next 24 hours...

**Comment by jorgebucaran** [comment]
~@okwolf Maybe let's hold for a while. →https://github.com/hyperapp/hyperapp/issues/610~

EDIT: Let's go <samp>@hyperapp/logger</samp>.

**Comment by jorgebucaran** [comment]
@okwolf Edited my comment above. Sorry for the never-ending package naming jumbling. I think it's decided, we are going to use @scoped packages.

### Related

- https://github.com/hyperapp/hyperapp/issues/610#issuecomment-367967118

**Comment by okwolf** [comment]
Closing since org/repo aren't changing after all.

**Comment by jorgebucaran** [comment]
😄npm i @hyperapp/logger

**Pull request closed without merging**